src/lib.rs

```diff
@@ -256,8 +285,93 @@ impl Navigator {
         }
     }
 
-    pub fn set_pwm_freq(&mut self) {
-        todo!()
+    /// Sets the pwm frequency of [`Navigator`].
+    ///
+    /// It changes the PRE_SCALE value on PCA9685.
+    ///
+    /// The prescaler value can be calculated for an update rate with the formula:
+    ///
+    /// `prescale_value = round(clock_freq / (4096 * desired_freq)) - 1`.
+    ///
+    /// The minimum prescaler value is 3, which corresonds to 1526 Hz.
+    /// The maximum prescaler value is 255, which corresponds to 24 Hz.
+    ///
+    /// If you want to control a servo, set a prescaler value of 100. This will
+    /// correspond to a frequency of about 60 Hz, which is the frequency at
+    /// which servos work.
+    ///
+    /// Internally this function stops the oscillator and restarts it after
+    /// setting the prescaler value if it was running.
```

Comment:
```suggestion
    /// Sets the PWM frequency of [`Navigator`].
    ///
    /// It changes the PRE_SCALE value on PCA9685.
    ///
    /// The prescaler value can be calculated for an update rate using the formula:
    ///
    /// `prescale_value = round(clock_freq / (4096 * desired_freq)) - 1`.
    ///
    /// The minimum prescaler value is 3, which corresponds to 1526 Hz.
    /// The maximum prescaler value is 255, which corresponds to 24 Hz.
    ///
    /// If you want to control a servo, set a prescaler value of 100. This will
    /// correspond to a frequency of about 60 Hz, which is the frequency at
    /// which servos work.
    ///
    /// Internally, this function stops the oscillator and restarts it after
    /// setting the prescaler value if it was running.
```